backup.html

```diff
@@ -0,0 +1,895 @@
+<!DOCTYPE html>
```

Comment:
- [ ] Backup files that you have, or files that are not related to the page should be ignored. Kindly create a file called .gitignore and type in the first line `backup.html`. This will not save the changes in a remote branch but you will have them locally on your machine.